app/assets/javascripts/controllers/middleware_topology/middleware_topology_controller.js

```diff
@@ -99,7 +99,16 @@ function MiddlewareTopologyCtrl($scope, $http, $interval, $location, topologySer
                 class_name = "middleware_deployment";
                 break;
             case "MiddlewareServer":
-                class_name = "middleware_server";
+                switch (d.item.product) {
+                    case "Hawkular":
+                        class_name = "vendor-hawkular";
```

Comment:
yes, I'm working on something similar in #7463 on icons, will upload a new version soon.
Regarding server types - if they are currently all "sort of Wildfly", but there will be others later on like vert.x or Fuse which are not Wildfly, maybe it makes sense to create a wildfly_server model to inherit from middleware_server? otherwise when adding others, they will not inherit "in parallel" to wildfly related, but from the one representing wildfly which might be not very correct. What do you think?